plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergSessionProperties.java

```diff
@@ -256,6 +259,18 @@ public IcebergSessionProperties(
                             validateMaxDataSize(PARQUET_WRITER_PAGE_SIZE, value, DataSize.valueOf(PARQUET_WRITER_MAX_PAGE_SIZE));
                         },
                         false))
+                .add(integerProperty(
+                        PARQUET_WRITER_PAGE_VALUE_COUNT,
+                        "Parquet: Writer page row count",
+                        parquetWriterConfig.getPageValueCount(),
+                        value -> {
+                            if (value < PARQUET_WRITER_MIN_PAGE_VALUE_COUNT || value > PARQUET_WRITER_MAX_PAGE_VALUE_COUNT) {
```

Comment:
nit: Maybe add another method in `io.trino.plugin.base.session.PropertyMetadataUtil` for between? There is already `validateMinDataSize` and `validateMaxDataSize`

Reply:
It might make sense to have dedicated method for data size, but it's integer here, explicit comparison seems simple enough.